bin/build

```diff
@@ -10,6 +10,9 @@ MIQ_ORG=${MIQ_ORG:-ManageIQ}
 SUI_ORG=${SUI_ORG:-ManageIQ}
 APPLIANCE_ORG=${APPLIANCE_ORG:-ManageIQ}
 
+NPM_REGISTRY_OVERRIDE=${NPM_REGISTRY_OVERRIDE:-""}
+NPM_REGISTRY_RESET=${NPM_REGISTRY_RESET:-false}
```

Comment:
Will we ever want to override and not reset? Can this just be one variable?

Reply:
"We" always want to reset, but not everyone. For example, if your build environment is same as where the images are used, you'd want to keep the non-default registry in the images so if you need to recompile assets, etc. it will continue to use non-default registry.

Reply:
@carbonin as per discussion, updated to always reset.